tests/unit/amo/helpers.js

```diff
@@ -124,6 +124,7 @@ export const fakeTheme = Object.freeze({
     version: '1.0',
   },
   type: ADDON_TYPE_THEME,
+  version: '0',
```

Comment:
This should have been:

```js
current_version: {
  ...fakeAddon.current_version,
  version: 0,
},
```

Reply:
Blarg, right, I'm a tool. I'll fix it up and merge it if it's green.

Reply:
Though it should be a string, not a number.